community_projects/index.html

```diff
@@ -0,0 +1,69 @@
+---
+layout: fullwidth
+title: Community Projects
+primary_title: Community Projects
+---
+
+<div id=intro>
+  <p>This page was made to highlight projects built by the community for the community.</p>
+  <p> Want your project on this page? To add your project to this page, please create a fork the <a href="https://github.com/opensearch-project/project-website" target="_blank" rel="noopener noreferrer">project website</a>, add your project to the <b>_community_projects</b> folder, and submit a pull request.</p>  
+</div>
+
+<table id="projects" >
+  <tr>
+    <th>Project Name</th>
+    <th>Project Owner</th>
+    <th>Description</th>
+    <th>License</th>
+    <th>Stars</th>
+    <th>Forks</th>
+  </tr>
+  {% for project in site.community_projects %}
```

Comment:
For a static site you probably want to do this as part of the build. It will also work better for SEO.

Reply:
https://github.com/18F/jekyll-get does a good job at this

(not sure if we're building the website or relying on GitHub pages to build it in which case custom plugins won't be available)

Reply:
The reason I took this approach is that that builds right now aren't frequent (unless @stockholmux I'm missing something). The GitHub data could get stale (stars, fork counts, etc). Since it is all data coming from GitHub i'm not sure if the tradeoff of putting this into the build process is worth risking having stale data. Thoughts?

Reply:
That makes sense. We don't have many projects listed now, so that's probably not an issue at build time (we could always add batching down the road). I can see if we have many people on the page it could be an issue with the current approach.

Reply:
It's more of a problem when bots start hitting the page (e.g. Google bot is very aggressive).

Reply:
I'm with @dblock - this will be a rate limiting problem. The rate limit will be based on the IP of the user - who knows what they are doing and certainly bots will have an exhausted limit. 

 I think we have a two options:
 1) Use github buttons https://buttons.github.io/
 2) Do this server side with a bash script that creates the yaml files and integrate it into the site build.

Reply:
@stockholmux Is this generated with GitHub pages or do you run a jekyll build? If it's the latter, just use that plugin I suggested instead of bash script with yaml.

Reply:
Unless I'm missing something the plugin only works well if you have **key <> json value** pairs. It doesn't look like it supports **key <> list of json values**. We have the latter; **community_projects <> [project 1, project 2, etc...]**. I took the bash script approach in my latest commit. It was not too complicated, but at build time the `generate.sh` script needs executed to import the data.